includes/amp.php

```diff
@@ -52,12 +52,11 @@ function amp_body_class( $classes ) {
  *
  * @param string   $item_output The menu item's starting HTML output.
  * @param WP_Post  $item        Menu item data object.
- * @param int      $depth       Depth of menu item. Used for padding.
  * @param stdClass $args        An object of wp_nav_menu() arguments.
  *
  * @return string Modified nav menu item HTML.
  */
-function amp_nav_sub_menu_buttons( $item_output, $item, $depth, $args ) {
```

Comment:
Should this be removed? We are not using the reference within the function. If we look at https://github.com/godaddy-wordpress/go/blob/ee70c2dfcafe7dfddc727c8f3c52e94898db966c/includes/core.php#L1213, for example, we can see the value properly used within a function.

Reply:
I would just ignore that line so it passes in the tests. I think we still need it present.